custom/shared/contexts/LiveStreamingProvider.js

```diff
@@ -0,0 +1,65 @@
+import React, {
+  createContext,
+  useContext,
+  useCallback,
+} from 'react';
+import { useLiveStreaming as useDailyLiveStreaming } from '@daily-co/daily-react-hooks';
+import PropTypes from 'prop-types';
+import { useUIState } from './UIStateProvider';
+
+export const LiveStreamingContext = createContext();
+
+export const LiveStreamingProvider = ({ children }) => {
+  const { setCustomCapsule } = useUIState();
```

Comment:
can you add a comment explaining this? Just looking at the code here and not looking at `useUIState`, it's not clear to me what custom capsule is referring to. We don't want to add comments where things are self-explanatory, but it's useful in spots where it might not be obvious what's happening without having a broader knowledge of the codebase